### ISSUE TITLE
InPost (IT, 1000+)

### ISSUE THREAD

**Issue by McCio** [alltheplaces/alltheplaces#11500]
### Brand name

InPost

### Wikidata ID

Q110970254

### Store finder url(s)

https://inpost.it/trova-un-locker
https://inpost.it/sites/default/files/points.json

### Sample store page url

* locker: https://inpost.it/locker-venezia-itven26716-viale-stazione
* point: https://inpost.it/punto-ritiro-itvve06079p-venezia-via-piave

### Countries?

IT

### Difficulty?

Medium - some customisation or data cleansing

### Number of POI?

1000+

### Behaviours

- [X] Has coordinates?
- [X] Has opening hours?
- [ ] JSON API?
- [X] Embedded JSON?
- [ ] HTML markup?
- [ ] Structured Data? (`pipenv run scrapy sd (specific page url)`) or [validator](https://validator.schema.org/) has content
- [ ] Open Graph Data?
- [ ] Sitemap? (`pipenv run scrapy sitemap (url)`)
- [ ] Requires proxy?
- [ ] Has 'bot protection' (Cloudflare, etc)?

**Comment by matkoniecz** [comment]
it may or may not help but data for inPost in Poland is already in ATP

maybe website structure is the same? Note that PL inpost dataset in ATP is a bit buggy

**Comment by McCio** [comment]
Yes, I had a quick look and the field names seem to be the same "shortened" things (`l`, `a`, `o`...) for the same meanings. The PL one has some additional cleanup specific for Poland language, but it might be harmonised in a country-specific step.
